src/main/scala/nl/knaw/dans/easy/fsrdb/FsRdbUpdater.scala

```diff
@@ -67,7 +92,7 @@ object FsRdbUpdater {
     XML.load(FedoraClient.getObjectXML(pid).execute().getEntityInputStream)
   }
 
-  private def getFileItem(pid: String)(objectXML: Elem)(implicit s: Settings): Try[FileItem] = Try {
+  private def getFileItem(datasetPid: String)(pid: String)(objectXML: Elem)(implicit s: Settings): Try[FileItem] = Try {
```

Comment:
When deciding whether to use currying or not, you should ask yourself if the partially applied functions that you can then get make any sense. I don't think they do here, except for the settings, which maybe even need to be curried.